spock-specs/src/test/groovy/org/spockframework/smoke/junit/UseJUnitTimeoutRule.groovy

```diff
@@ -32,7 +32,7 @@ class UseJUnitTimeoutRule extends EmbeddedSpecification {
   }
 
   def "feature method that completes in time"() {
-    timeout = 500
+    timeout.set(5000)
```

Comment:
Same answer: we have slow Shippable integration. It does not work find in parallel mode. 250ms is usual delay for thread switching on Shippable boxes.